src/program/mod.rs

```diff
@@ -548,6 +552,12 @@ impl<'a> Program<'a> {
                     _ => {}
                 }
             }
+            if let Some(ts) = self.ui_state.rerender_time {
+                if Instant::now().duration_since(ts) > Duration::from_secs(2) {
```

Comment:
Are success messages meant to be read fully, or glanced at that the operation was successful.
2 seconds is enough to read it fully. One second is enough to glance at it.